rabbitmq/metadata.csv

```diff
@@ -29,9 +29,9 @@ rabbitmq.queue.active_consumers,gauge,,,,"Number of active consumers, consumers
 rabbitmq.queue.bindings.count,gauge,,,,Number of bindings for a specific queue,0,rabbitmq,queue bind,
 rabbitmq.queue.consumers,gauge,,,,Number of consumers,0,rabbitmq,cons,
 rabbitmq.queue.consumer_utilisation,gauge,,fraction,,The ratio of time that a queue's consumers can take new messages,0,rabbitmq,cons util,
-rabbitmq.queue.head_message_timestamp,gauge,,millisecond,,"[Management CLI] Timestamp of the head message of the queue [OpenMetricsV2] Timestamp of the first message in the queue, if any",0,rabbitmq,head msg ts,
+rabbitmq.queue.head_message_timestamp,gauge,,millisecond,,"[OpenMetrics, Management CLI] Timestamp of the head message of the queue Timestamp of the first message in the queue, if any",0,rabbitmq,head msg ts,
 rabbitmq.queue.memory,gauge,,byte,,"Bytes of memory consumed by the Erlang process associated with the queue, including stack, heap and internal structures",0,rabbitmq,mem,
-rabbitmq.queue.messages,gauge,,message,,[Management CLI] Count of the total messages in the queue [OpenMetricsV2] Sum of ready and unacknowledged messages - total queue depth,0,rabbitmq,msgs,
+rabbitmq.queue.messages,gauge,,message,,[OpenMetrics, Management CLI] Count of the total messages in the queue Sum of ready and unacknowledged messages - total queue depth,0,rabbitmq,msgs,
```

Comment:
Not sure if you're aware, but this line is invalid and is causing the metrics metadata sync task to fail -- I think this should be in `""`

Reply:
@alai97